projects/angular/src/layout/API.md

```diff
@@ -0,0 +1,55 @@
+# Card - Style API
+
+## CSS Variables
+
+| CSS Variable Name              | Description                                                                              |
+| ------------------------------ | ---------------------------------------------------------------------------------------- |
+| --clr-card-bg-color            | Changes the background color of card.  |
+| --clr-card-divider-color       | Changes the divider color of the card.                              |
+| --clr-card-title-color         | Changes the text color of card header.           |
+| --clr-card-title-font-weight      | Changes the thickness of card title.                        |
+| --clr-card-border-width | Changes the border width of the card. |
+| --clr-card-border-radius | Changes the border radius of card. |
+| --clr-card-border-color | Changes the border color of the card. |
+| --clr-card-box-shadow-color | Changes the background shadow color of card. |
+| --clr-card-clickable-border-color |  Changes the background color of card when clicked.|
+| --clr-card-clickable-box-shadow-color | Changes the background shadow color of card when clicked.|
+
+
+## Class names
+
+| Class name | Description                                                    |
+| ---------- | -------------------------------------------------------------- |
+| clr-card   | It usually has a rectangle, image, title, description, and links. CSS arranges cards for design flexibility. |
+
+
+
+# Login - Style API
+
+## CSS Variables
+
+| CSS Variable Name              | Description                                                                              |
+| ------------------------------ | ---------------------------------------------------------------------------------------- |
+| --clr-login-title-color            | Changes the color of header in login page.                                |
+| --clr-login-title-font-weight       | Changes the title thickness of login page.                      |
+| --clr-login-title-font-family        | Changes the font style of title in login page.           |
+| --clr-login-trademark-color      | Changes the color of trademark logo.                        |
+| --clr-login-trademark-font-weight | Changes the thickness of trademark logo in login page. |
+| --clr-login-trademark-font-family | Changes the logo style of trademark icon in the login section. |
+| --clr-login-subtitle-color | Changes the text color of the login page's sub title. |
+| --clr-login-subtitle-font-weight | Changes the thickness of the login page sub title. |
+| --clr-login-subtitle-font-family | Changes the style of the login page sub title.|
+| --clr-login-background-color | Changes the background color of the login page.|
+| --clr-login-error-background-color | Changes the background color of login page's to indicate errors.|
```

Comment:
more like "background color of the error message"

Reply:
Changed the description as per the suggestions.